help/en/html/hardware/cmri/CMRI.shtml

```diff
@@ -970,37 +978,37 @@ for easy access click an item in this list:</p>
   the node type (NDP) of C which is a cpNode. DL is the delay value, and Opts
   are node options entered in the node configuration window. A
   <strong>Transmit</strong> message follows with the node address (ua) of 20.
-  OB are the output data bytes as hexadecimal values. Five polls are seen
+  OB are the output data bytes as hexadecimal values. Polls are seen
   followed by the poll/response sequence.</p>
 
   <p><img alt="" src="images/cmrinetmonitor-pollinitialization.png"
-  width="1040" height="586"></p>
-
+  width="904" height="559"></p>
+</ul>
```

Comment:
See comments above about allowed list contents.

Reply:
Corrected lines in error.